html/pfappserver/root/graph/dashboard.tt

```diff
@@ -747,6 +748,7 @@ END
                     </div>
                 </div>
             </div>
+            [% END %]
```

Comment:
We could add a "ELSE" statement like this:
```html
<h2>[% l('DHCP on') %] [% network %] [% l('disabled') %]</h2>
```
to avoid a white page if all your DHCP servers are disabled.

Reply:
Before `[% END %]`
```html
            [% ELSE %]
            <div class="card">
                <div class="card-title">
                    <h2>[% l('DHCP on') %] [% network %] [% l('disabled') %]</h2>
                </div>
            </div>
            [% END %]
```